_temp/hud/src/sections/section-ajax.js

```diff
@@ -7,128 +7,148 @@ var process = require('./util/process');
+    recordItem('<tbody class="' + rowClass + '"><tr><td class="glimpse-hud-listing-overflow" title="' + uri + '" colspan="2">' + clickableUri + '</td><td class="glimpse-hud-listing-value glimpse-data-duration">' + duration + '</td><td class="glimpse-hud-listing-value glimpse-data-size">' + processSize(size) + '</td></tr><tr><td class="glimpse-hud-quite glimpse-data-content-method">' + method + '</td><td class="glimpse-hud-quite glimpse-hud-listing-overflow">' + status + ' - ' + statusText + '</td><td class="glimpse-hud-quite glimpse-data-content-type glimpse-hud-listing-overflow" title="' + contentType + '">' + processContentType(contentType) + '</td><td class="glimpse-hud-quite glimpse-data-content-time">' + time.toTimeString().replace(/.*(\d{2}:\d{2}:\d{2}).*/, "$1") + '</td></tr></tbody>', '.glimpse-hud-section-ajax .glimpse-data-ajax-detail', detailStack, 6);
 };
 var recordItem = function(html, selector, stack, length) {
-	//Set row
-	var row = $(html).prependTo(selector);
-	setTimeout(function() {
-		row.addClass('added');
-	}, 1);
-
-	//Track state of the details
-	if (stack.length >= length)
-		stack.shift().remove();
-	stack.push(row);
+    //Set row
+    var row = $(html).prependTo(selector);
+    setTimeout(function() {
+        row.addClass('added');
+    }, 1);
+
+    //Track state of the details
+    if (stack.length >= length)
+        stack.shift().remove();
+    stack.push(row);
 };
 
 var render = function(details, opened) {
-	process.init(structure);
+    process.init(structure);
 
-	return rendering.section(structure, details, opened);
+    return rendering.section(structure, details, opened);
 };
 var postRender = function() {
-	var open = XMLHttpRequest.prototype.open; 
-	XMLHttpRequest.prototype.open = function(method, uri) {
-		if (util.isLocalUri(uri) && uri.indexOf('/glimpse/') == -1) { 
-			this._uri = uri;
-			
-			var startTime = new Date().getTime(); 
-			this.addEventListener('readystatechange', function() {
-					if (this.readyState == 4 && this.getResponseHeader('__glimpse-id'))  { 
-						update(method, uri, new Date().getTime() - startTime, this.getResponseHeader('Content-Length'), this.status, this.statusText, new Date(), this.getResponseHeader('Content-Type'), this.getResponseHeader('__glimpse-id'));
-					}
-				}, false); 
-		}
-
-		open.apply(this, arguments);
-	};    
-	
-	var send = XMLHttpRequest.prototype.send; 
-	XMLHttpRequest.prototype.send = function() {
-		if (this._uri){
-			this.setRequestHeader('__glimpse-isAjax', true);
-		}
-		
-		send.apply(this, arguments);
-	};                      
+    ready = true;
+
+    for (var task in preRenderCache) {
+        task();
+    }
+    preRenderCache = undefined;
```

Comment:
do you just want to call `clear()` on this instead of setting it to undefined?  I'm not clear on where/how this gets re-assigned, so I'm not clear that this is safe from null references.